frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/GoService.java

```diff
@@ -68,12 +88,26 @@ public void onCreate() {
         // focus. This is needed to avoid timeouts when the backend is polling the BitBox for e.g.
         // an address verification.
         startForeground(notificationId, notification);
+
+        // Register USB broadcast receiver to detect USB disconnects, even while the app is in the
+        // background.
+        IntentFilter filter = new IntentFilter();
+        filter.addAction(UsbManager.ACTION_USB_DEVICE_DETACHED);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
The version code ('O' -> 26) seems wrong. The RECEIVER_EXPORTED should be available since version 33 (https://developer.android.com/reference/android/content/Context#RECEIVER_EXPORTED)

Reply:
Good catch, I had only looked at registerReceiver itself:

https://developer.android.com/reference/android/content/Context#registerReceiver(android.content.BroadcastReceiver,%20android.content.IntentFilter,%20int)

Fixed.

Reply:
I noticed it as it was a warning in my Android Studio